third_party/move/move-vm/runtime/src/native_functions.rs

```diff
@@ -146,18 +149,25 @@ impl<'a, 'b> NativeContext<'a, 'b> {
         &mut self,
         address: AccountAddress,
         ty: &Type,
-    ) -> VMResult<(bool, Option<NumBytes>)> {
-        // TODO(Rati, George): propagate exists call the way to resolver, because we
-        //                     can implement the check more efficiently, without the
-        //                     need to actually load bytes.
-        let (value, num_bytes) = self
-            .data_store
-            .load_resource(self.module_storage, self.resource_resolver, address, ty)
-            .map_err(|err| err.finish(Location::Undefined))?;
-        let exists = value
-            .exists()
-            .map_err(|err| err.finish(Location::Undefined))?;
-        Ok((exists, num_bytes))
+    ) -> PartialVMResult<(bool, Option<NumBytes>)> {
+        // TODO(gelash, georgemitenkov):
```

Comment:
I'd prefer to use a bug number for each TODO. It could be a generic bug used in multiple places. This makes it easier to track.

Reply:
Done